packages/sit-onyx/src/components/OnyxStepper/OnyxStepper.vue

```diff
@@ -84,9 +84,33 @@ const getFormattedValue = computed(() => {
     }
   };
 });
+const getDisplayValue = computed(() => {
+  return (value: Nullable<number>) => {
+    if (value == undefined) return "";
+    if (props.formatNumber) {
+      if (typeof props.formatNumber === "boolean") {
+        if (props.precision) {
+          return value.toLocaleString(undefined, {
+            minimumFractionDigits: props.precision,
+            maximumFractionDigits: props.precision,
+          });
+        }
+        return value.toLocaleString();
```

Comment:
We should use the `locale` from `injectI18n` to use the user language:

```suggestion
          return value.toLocaleString(locale.value, {
            minimumFractionDigits: props.precision,
            maximumFractionDigits: props.precision,
          });
        }
        return value.toLocaleString(locale.value);
```